pixi.toml

```diff
@@ -0,0 +1,67 @@
+# Pixi is a package management tool for developers.
+# Before running a task, pixi ensures that all listed dependencies are installed first.
+#
+# Run the tasks with `pixi run TASK`, e.g. `pixi run codegen`.
+# Extra CLI argument after `pixi run TASK` are passed to the task cmd.
+# Lists tasks with `pixi task list`
+#
+# Type `pixi shell` to enter the pixi environment.
+#
+# https://prefix.dev/docs/pixi/overview
+
+[project]
+name = "rerun"
+authors = ["Jeremy Leibs <jeremy@rerun.io>"]
+channels = ["conda-forge"]
+description = "Log images, point clouds, etc, and visualize them effortlessly"
+homepage = "https://rerun.io"
+license = "MIT OR Apache-2.0"
+platforms = ["linux-64", "osx-arm64", "osx-64", "win-64"]
+readme = "README.md"
+repository = "https://github.com/rerun-io/rerun"
+version = "0.1.0"                                                              # TODO(emilk): sync version with `Cargo.toml` with help from `crates.py`
+
+
+[tasks]
+# Note: extra CLI argument after `pixi run TASK` are passed to the task cmd.
+
+# Run the codegen. Optionally pass `--profile` argument if you want.
+codegen = "cargo run --package re_types_builder -- "
+
+py-build = "maturin develop --manifest-path rerun_py/Cargo.toml --extras=tests"
+py-test = { cmd = "python -m pytest -vv rerun_py/tests/unit", depends_on = [
+  "py-build",
+] }
+
+cpp-prepare = "cmake -G Ninja -B build -S . -DCMAKE_BUILD_TYPE=Debug"
```

Comment:
note to self: Check if this still works fine on windows (since pixi should download ninja for windows it might as well work fine, opening the door to more unified build experience than what I had in mind! \o/)

Reply:
There's this in CMakeLists.txt right now but I think we still need it for the time being on CI
```
# Use makefiles on linux, otherwise it might use Ninja which isn't installed by default.
if(NOT DEFINED CMAKE_GENERATOR AND UNIX)
    set(CMAKE_GENERATOR "Unix Makefiles")
endif()
```
is to be removed!